samples/quantum-approximation-optimization/qaoa-for-quadratic-unconstrained-binary-optimization.ipynb

```diff
@@ -0,0 +1,819 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QAOA and QAA to solve a QUBO problem"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "metadata": {},
+   "source": [
+    "# Restart the kernel once you have run this cell\n",
+    "!pip install -U -q azure-quantum\n",
+    "\n",
+    "try:\n",
+    "    import pulser\n",
+    "except ImportError:\n",
+    "    # This installs two python packages:\n",
+    "    #   - pulser (core package)\n",
+    "    #   - pulser_simulation\n",
+    "    # If you don't need simulation in your own workflow, you may run instead:\n",
+    "    #   !pip -q install pulser-core\n",
+    "    # https://pulser.readthedocs.io/en/stable/installation.html\n",
+    "    !pip -q install pulser"
```

Comment:
```suggestion
    "    import pulser\n",
    "    import pulser_simulation\n",
    "except ImportError:\n",
    "    # This installs two python packages:\n",
    "    #   - pulser (core package)\n",
    "    #   - pulser_simulation\n",
    "    # If you don't need simulation in your own workflow, you may run instead:\n",
    "    #   !pip -q install pulser-core\n",
    "    # https://pulser.readthedocs.io/en/stable/installation.html\n",
    "    !pip -q install --upgrade pulser"
```
With an old pulser version on my Quantum Worskpace, I experienced an import error in the next cell.
Just added a pulser_simulation import in the try block, and the `--upgrade` parameter to the pip command.

Reply:
Great ! Thanks for having investigated and solved that.